algorand/MEMORY.md

```diff
@@ -217,6 +221,7 @@ To see what the layout looks like, let’s decode the first few bytes of the byt
 
 and the first few lines of the TEAL code as a reminder:
 
+<!-- cspell:enable -->
```

Comment:
is this inverse with the one below?

Reply:
Fixed in bc975ceec9c476a9cb57871fbb1d961805a1f647